### PR TITLE
fix(backend): enhance nodeinfo by export instance admin via nodeAdmins key

### PR DESCRIPTION
**Pull request by 6543**
https://codeberg.org/thefederationinfo/nodeinfo_metadata_survey

### CODE REVIEW

**Comment by 6543** [comment]
followup of #11830

File: packages/backend/src/server/NodeinfoServerService.ts [hunk 1]
```diff
@@ -96,6 +96,11 @@ export class NodeinfoServerService {
 				metadata: {
 					nodeName: meta.name,
 					nodeDescription: meta.description,
+					nodeAdmins: [{
+						name: meta.maintainerName,
+						email: meta.maintainerEmail,
+					}],
+					// deprecated
```
Comment (acid-chicken) [inline packages/backend/src/server/NodeinfoServerService.ts:103]: Can you please provide a source?

Comment (6543) [reply packages/backend/src/server/NodeinfoServerService.ts:103]: if used internally its not deprecated else we should do so.

I try to stream line the nodeinfo across different federated projects.
the endgame is to release a new nodeinfo spec ... but that will take it's time.

If there is interest just visit https://github.com/jhass/nodeinfo/issues/70 ...

**Comment by 6543** [comment]
@acid-chicken it's also fin to not depricate it ... just tell :)

**Review by acid-chicken — APPROVED** [review]

**Comment by syuilo** [comment]
🙏

**Pull request merged**